src/ui/views/app.tsx

```diff
@@ -1,10 +1,11 @@
-import React, { useState, useEffect, lazy, Suspense } from 'react'
-import { Box, Text, useApp, useInput, useStdout } from 'ink'
+import React, { useState, useEffect } from 'react'
+import { Box, Newline, Text, useApp, useInput, useStdout } from 'ink'
 import Logo from '../components/logo'
-import DockerLogs from '../components/dockerlogs'
-import Select from '../components/selectInput'
-
-const Terminal = lazy(() => import('../components/terminal'))
+import SelectInput from 'ink-select-input'
+import NodeStatus from '../components/status'
+import NodeInfo from '../components/nodeInfo'
+import PeerInfo from '../components/peerInfo'
+import nodeJson from '../services/nodeJson.json'
```

Comment:
it should be a dynamic setting, so we should read it from fs in `~.bdk` not in the code dir